api/src/pcapi/routes/backoffice/templates/layouts/connected.html

```diff
@@ -153,6 +153,7 @@ <h6 class="mdc-deprecated-list-group__subheader">{{ name }}</h6>
                   {% if get_setting("ENABLE_TEST_USER_GENERATION") %}
                     {{ menu_section_item("Générateur d'utilisateurs de test", "backoffice_web.generate_user") }}
                   {% endif %}
+                  {% if get_setting("ENABLE_USER_DELETION") %}{{ menu_section_item("Suppression d'utilisateurs", 'backoffice_web.delete_user') }}{% endif %}
```

Comment:
Un seul à la fois :)
```suggestion
                  {% if get_setting("ENABLE_USER_DELETION") %}{{ menu_section_item("Suppression d'utilisateur", 'backoffice_web.delete_user') }}{% endif %}
```